iceberg-handler/pom.xml

```diff
@@ -0,0 +1,189 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>org.apache.hive</groupId>
+        <artifactId>hive</artifactId>
+        <version>4.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>iceberg-handler</artifactId>
+    <packaging>jar</packaging>
+    <name>Hive Iceberg Handler</name>
+
+    <properties>
+        <hive.path.to.root>..</hive.path.to.root>
+        <iceberg-api.version>0.11.0</iceberg-api.version>
+        <kryo-shaded.version>4.0.2</kryo-shaded.version>
+        <avro.iceberg.version>1.9.2</avro.iceberg.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.iceberg</groupId>
+            <artifactId>iceberg-api</artifactId>
+            <version>${iceberg-api.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.iceberg</groupId>
+            <artifactId>iceberg-core</artifactId>
+            <version>${iceberg-api.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.iceberg</groupId>
+            <artifactId>iceberg-hive-metastore</artifactId>
+            <version>${iceberg-api.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.iceberg</groupId>
+            <artifactId>iceberg-data</artifactId>
+            <version>${iceberg-api.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.iceberg</groupId>
+            <artifactId>iceberg-parquet</artifactId>
+            <version>${iceberg-api.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.iceberg</groupId>
+            <artifactId>iceberg-orc</artifactId>
+            <version>${iceberg-api.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-client</artifactId>
+            <version>${hadoop.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.avro</groupId>
+                    <artifactId>avro</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.hive</groupId>
+            <artifactId>hive-exec</artifactId>
+            <version>${project.version}</version>
+            <exclusions>
+                <exclusion>
```

Comment:
Why do we have these exclusions?

Reply:
The goal was to create the same dependency structure as much as possible as the original one: https://github.com/apache/iceberg/blob/master/build.gradle#L589

Reply:
I would try to focus on keeping the same source files as in the Iceberg repo, so we can easily port changes between the 2, but otherwise I would not try to stick to the same things just because it was the same there.

OTOH if we know the reason why they were removed and it applies here too, then we should do the same

Reply:
Makes sense. As discussed offline, I've excluded orc, parquet, avro, guava and fasterxml only